src/app/navbar/navbar.component.ts

```diff
@@ -7,8 +8,29 @@ import { Component, OnInit } from '@angular/core';
 })
 export class NavbarComponent implements OnInit {
 
-  constructor() { }
+  constructor(private router: Router) { }
 
+  // Check if User is Logged In
+  isLoggedin(){
+    const token_exists = 'token' in localStorage;
```

Comment:
Use `localStorage.hasOwnProperty(key)` or `localStorage.getItem(key)`

`in` is not a recommended way since it looks up in the prototype chain and hence can give false positives in case of keys like `length`